packages/@uppy/core/types/index.d.ts

```diff
@@ -56,29 +70,47 @@ export interface Store {
   subscribe(listener: any): () => void;
 }
 
+interface LocaleObject {
+  [key: string]: string | LocaleObject;
```

Comment:
A locale object can only have a `strings` and `pluralize` key, I think it should be:

```ts
interface LocaleStrings {
  [key: string]: string | LocaleStrings;
}
interface Locale {
  strings: LocaleStrings;
  pluralize: (n: number) => number;
}
```